eidr-connect.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+ethernet="eth0"
+
+if [[ $1 && $2 ]]; then
+  if [ "$1"=="--ethernet" ]; then
+    ethernet="$2"
+  fi
+fi
+
+./initial-checks.sh --ethernet $ethernet || exit 1
+
+export REPO_ROOT=$(pwd) 
+
+#Ensure data dump file is in our directory
+aws s3 cp s3://bsve-integration/spa-mongodump.tar ./spa-mongodump.tar 
+
+#Build and spin up our mongodb
+./mongodb.sh --ethernet $ethernet
+
+#Import the spa mongo data
```

Comment:
This looks redundant since the ProMED scraper also loads a dump of our ProMED database.